### PR TITLE
Issue #126 - Gracefully fail when database connection can't be made for playback

### LINKED ISSUE #126: Telemetry Playback component and server endpoints should fail gracefully
The playback components and server endpoints should fail gracefully if a database connection cannot be made. At the moment a failed database connection will prevent the GUI plugin from booting properly.

### PR DESCRIPTION
**Pull request by MJJoyce**
Update the server endpoints and Playback component so they fail gracefully when a database connection cannot be made.

Resolve #126

### CODE REVIEW

File: ait/gui/__init__.py [hunk 1]
```diff
@@ -127,21 +127,31 @@ class Playback(object):
     A Playback manages the state for the playback component.
     playback.dbconn: connection to database
     playback.query: time query map of {timestamp: list of (uid, data)} from database
-    playback.on: true if gui is currently in playback mode
+    playback.on: True if gui is currently in playback mode. Real-time telemetry will not
+        be sent to the frontend during this.
```
Comment (aywaldron) [inline ait/gui/__init__.py]: Just curious is stopping real-time telem the intended functionality, or something to change in the future?

Comment (MJJoyce) [reply ait/gui/__init__.py]: That's intended. If we're playing back telemetry from a historical query we cut off the real time stream to the clients so the older data can be replayed.

**Comment by aywaldron** [comment]
When I test this without a database connection nothing shows up under the playback tab, should we add a statement saying playback is disabled?

**Comment by MJJoyce** [comment]
@aywaldron, did you build the static files? At the moment they're out of date in this PR so you won't see it by default. I planned to update master after we merge so we can be up-to-date with stuff.

**Comment by aywaldron** [comment]
@MJJoyce Ok that explains it, since I saw the statement in the code. Sounds good then.

**Comment by aywaldron** [comment]
@MJJoyce Is the build with `npx webpack`? I still don't see the statement after doing that.

**Pull request merged**